code/index.html

```diff
@@ -3,28 +3,98 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <title>Project 3: Business Site</title>
+    <link rel="icon" href="./purple.png"/>
     <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap" rel="stylesheet">
   </head>
   <body>
     <!-- video or image as a header is cool :) -->
+    <header>
+      <video class="hero-video" autoplay loop muted playsinline>
+        <source src="./videos/pexels-ivan-khmelyuk-8642288-720p.mp4" type="video/mp4">
+        Your browser does not support the video tag.
+      </video>
+      <div class="overlay-text">
+        <h1 class="top-text">Visit our relaxing purple resort</h1>
+        <h2 class="top-text-b">How lavender can make you calm and relax</h2>
+      </div>
+    </header>
 
-    <!-- Signup form -->
+    <section class="grid">
+      <!-- Signup form -->
+      <section class="form-container">
+        <form action="http://httpbin.org/anything" method="POST"> <!-- Posts form to httpbin -->
+          <h2>Book today!</h2>
+          <!-- Form boxes here -->
```

Comment:
I really like these small comments on what comes next sort of! Makes it easy to get a picture of how the site is structured!